packages/react/src/components/Forms/Switch/index.scss

```diff
@@ -0,0 +1,89 @@
+.c__switch {
+
+  input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+    margin: 0;
+    // This is made to prevent a bug on Chromium were was the labelSide is set to right then it
+    // creates an artificial margin between successive switches.
+    position: absolute;
+  }
+
+  .c__checkbox__container {
+    justify-content: space-between;
+  }
+
+  input:checked + &__rail {
+    background-color: var(--c--components--forms-switch--accent-color);
+  }
+
+  input:checked + &__rail:before {
+    transform: translateX(20px);
+  }
+
+  &__rail__wrapper {
+    display: inline-flex;
+  }
+
+  &__rail {
+    position: relative;
+    cursor: pointer;
```

Comment:
Should we add `cursor: pointer` to the whole component like you did in PR #71 for radio and checkbox ?